autotx/tests/agents/token/test_swap.py

```diff
@@ -1,7 +1,10 @@
 from autotx.utils.ethereum import load_w3
+from autotx.utils.ethereum.lifi.swap import SLIPPAGE
 from autotx.utils.ethereum.networks import NetworkInfo
 from autotx.utils.ethereum.eth_address import ETHAddress
 
+PLUS_DIFFERENCE_PERCENTAGE = 0.011
```

Comment:
Shouldn't this be slippage?

Reply:
no, because the output amount is not always exactly equal to what the user requests. this means that, if the user requests 1000 USDC, the minimum amount could be 1000.7 - this might make the user to receive 1005.7 (being more than the slippage)

i will update this amount to be `0.007` so it is closer to slippage of swaps